src/CredentialsCheckFlood.php

```diff
@@ -123,7 +123,7 @@ protected function getAccountIdentifier($ip, $name) {
       $account_by_name = $storage->loadByProperties(['name' => $name]);
       $this->accounts[$name] = reset($account_by_name);
     }
-    /** @var \Drupal\Core\Session\AccountInterface $account **/
+    // @var \Drupal\Core\Session\AccountInterface $account
```

Comment:
Hi, this should work:

Instead of:
/** @var \Drupal\Core\Session\AccountInterface $account **/
it should be a single star at end:
/** @var \Drupal\Core\Session\AccountInterface $account */